HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -17,6 +17,24 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+# Eta Extended Electrons 
+def customiseForPRNUM(process):
```

Comment:
```suggestion
def customiseFor35309(process):
```

Reply:
Please change `PRNUM` to `35309`, here and below (only a technicality, but necessary..).